community/c++-algo/include/GameLib/src/navigation.h

```diff
@@ -0,0 +1,56 @@
+/*
+Description: A header for the path finder.
+Last Modified: 08 Apr 2019
+Author: Patrik Dobias
+*/
+
+#ifndef NAVIGATION_H
+#define NAVIGATION_H
+
+#include <limits>
+#include <algorithm>
+#include <vector>
+#include <queue>
+
+#include "GameLib/src/util.h"
+#include "GameLib/src/structs.h"
+#include "GameLib/src/gameState.h"
```

Comment:
It is clearer if you simply include without the path (since they are in the same directory):
```
#include "util.h"
#include "structs.h"
#include "gameState.h"
```